runtime/plugin/tutor-mode.vim

```diff
@@ -0,0 +1,12 @@
+if exists('g:loaded_tutor_mode_plugin') || &compatible
```

Comment:
Why is this file called `tutor-mode.vim` instead of `tutor.vim` like the others?